### PR TITLE
Switch to Python 3.7 and use Pipenv

### LINKED ISSUE #7: Use pipenv
https://docs.pipenv.org/

### PR DESCRIPTION
**Pull request by filipelbc**
Closes #7

### CODE REVIEW

File: todo/Pipfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+pylint = "*"
+autopep8 = "*"
+pep8-naming = "*"
+
+[packages]
+django = "*"
+djangorestframework = "*"
+
+[requires]
+python_version = "3.6"
```
Comment (danielmoraes) [inline todo/Pipfile]: I am using Python 3.7 on Arch Linux. Either a specify minimum Python version or remove this constraint.

File: todo/README.md [hunk 2]
```diff
@@ -1,28 +1,30 @@
...
 
 ## How to Run
 
-To run locally, do:
+To run the development server, do:
 
 - Create the Python3 virtual environment
   ```shell
-  make venv
+  pipenv install
```
Comment (danielmoraes) [inline todo/README.md]: Use `make venv` instead.

File: todo/README.md [hunk 3]
```diff
@@ -1,28 +1,30 @@
...
   ```
 
 - Run the development server
   ```shell
+  pipenv shell
   ./manage.py runserver
   ```
 
-- Test the API endpoints
+- Test the API endpoints (using [https://httpie.org/](Httpie))
```
Comment (danielmoraes) [inline todo/README.md]: The link name comes before the URL: `[name](url)`.

The name of the library is HTTPie, not Httpie.

File: todo/README.md [hunk 4]
```diff
@@ -1,28 +1,30 @@
 # TODO Server
 
-The server is a Django project.
+The server is a [https://www.djangoproject.com/](Django) project. It uses
+Python 3.6 and [https://pipenv.readthedocs.io/en/latest/](Pipenv).
```
Comment (danielmoraes) [inline todo/README.md]: The link name comes before the URL: `[name](url)`.

File: todo/README.md [hunk 5]
```diff
@@ -1,28 +1,30 @@
...
   ```shell
-  make venv
+  pipenv install
   ```
 
 - Create development database, migrate and add initial data
   ```shell
-  . venv/bin/activate
+  pipenv shell
   make
```
Comment (danielmoraes) [inline todo/README.md]: The second command depends on the first one:
```shell
pipenv shell && make
```

File: todo/README.md [hunk 6]
```diff
@@ -1,28 +1,30 @@
...
   ```
 
 - Run the development server
   ```shell
+  pipenv shell
   ./manage.py runserver
```
Comment (danielmoraes) [inline todo/README.md]: Same as before.

File: todo/Pipfile (hunk 1 above)
Comment (filipelbc) [reply todo/Pipfile]: You can run arbitrary versions of Python using pyenv. I've update it.

File: todo/README.md (hunk 2 above)
Comment (filipelbc) [reply todo/README.md]: Done

File: todo/README.md (hunk 3 above)
Comment (filipelbc) [reply todo/README.md]: Fixed

File: todo/README.md (hunk 4 above)
Comment (filipelbc) [reply todo/README.md]: Fixed

File: todo/README.md (hunk 5 above)
Comment (filipelbc) [reply todo/README.md]: Changed

File: todo/README.md (hunk 6 above)
Comment (filipelbc) [reply todo/README.md]: Changed

**Pull request merged**